app/assets/javascripts/discourse/app/components/small-user-list.gjs

```diff
@@ -38,38 +29,51 @@ export default class SmallUserList extends Component {
   }
 
   <template>
-    {{#each this.users as |user|}}
-      {{#if user.unknown}}
-        <div
-          title={{i18n "post.unknown_user"}}
-          class="unknown"
-          role="listitem"
-        ></div>
-      {{else}}
-        <a
-          class="trigger-user-card"
-          data-user-card={{user.username}}
-          title={{user.username}}
-          aria-hidden="false"
-          role="listitem"
+    <PluginOutlet
+      @name="small-user-list-internal"
+      @outletArgs={{hash data=this.args}}
+    >
+      <div class="clearfix small-user-list" ...attributes>
```

Comment:
This looks like a change in HTML structure, which could affect plugins/themes. Is it absolutely necessary for the backport?

Reply:
This div is not, the span under it contains a `role="list"` and without it I get an error `You have an element with the role of "listitem" but it is missing the required (immediate) parent element of "[group, list]` so I added this from the main branch.

Reply:
Is the error from ember-template-lint? Or the browser? (if it's the former, we could just add a `{{! template-lint-ignore}}` directive